test/lib/server.c

```diff
@@ -62,7 +62,13 @@ void test_server_start(struct test_server *s, const MunitParameter params[])
 {
 	int rv;
 
-	rv = dqlite_node_create(s->id, s->address, s->dir, &s->dqlite);
+	bool disk_mode = false;
+	const char *disk_mode_param = munit_parameters_get(params, "disk_mode");
+	if (disk_mode_param != NULL) {
+		disk_mode = (bool)atoi(disk_mode_param);
+	}
```

Comment:
[Nit]: What about?
```c
const char *disk_mode_param = munit_parameters_get(params, "disk_mode");
int disk_mode = disk_mode_param != NULL ? (bool)atoi(disk_mode_param) : false;
```

Reply:
Better indeed!

Reply:
I ended up factoring this out into a function since it's used frequently.